tests/bluetooth/mesh_shell/testcase.yaml

```diff
@@ -3,9 +3,7 @@ common:
   harness: keyboard
 tests:
   bluetooth.mesh.mesh_shell:
-    platform_allow: qemu_x86 nrf51dk_nrf51422
-    integration_platforms:
-      - qemu_x86
+    platform_allow: qemu_cortex_m3 qemu_x86 nrf51dk_nrf51422 nrf52dk_nrf52832
```

Comment:
Seems this unnecessary change sneaked somehow. The initial upstream file is more correct. I guess changes in this file should be reverted.